spacy/lang/hi/lex_attrs.py

```diff
@@ -15,16 +15,17 @@
 ]
 # fmt: on
 
-# reference 1:https://en.wikipedia.org/wiki/Indian_numbering_system
+# reference 1: https://en.wikipedia.org/wiki/Indian_numbering_system
 # reference 2: https://blogs.transparent.com/hindi/hindi-numbers-1-100/
+# reference 3: https://www.mindurhindi.com/basic-words-and-phrases-in-hindi/
 
 _num_words = [
     "शून्य",
     "एक",
     "दो",
     "तीन",
     "चार",
-    "पांच",
+    "पांच", "पाँच",
```

Comment:
This is a good opportunity to introduce a convention - the first word is primary, as per literature grade Hindi, and rest are acceptable variations.
This comment also applies to other such pairs in this list.

Reply:
Bindu and Chandra Bindu both are widely being used interchangeably informal news reporting such as BBC and Navabharat times and applies to all the words in Hindi such as चाँद, मुँह, हँसना, दाँत, etc. 

Check this out: https://groups.google.com/forum/#!topic/hindianuvaadak/cZf1OvF-Fck

I think a long-term solution will be to process this for all words in normalization step.